gslib/tests/test_signurl.py

```diff
@@ -22,6 +22,7 @@
 from datetime import datetime
 from datetime import timedelta
 import pkgutil
+from six import PY2, PY3
```

Comment:
For consistency's sake (with the rest of the code base), can we just import `six` and reference these below as `six.PY2` and `six.PY3`?

Reply:
Sounds good!

Reply:
After removing the extra hashes, PY2 and PY3 weren't used at all in this file anymore, so I just removed the import entirely.